cmeutils/tests/test_plot.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+import scipy.signal as signal
```

Comment:
```suggestion
```
Here we don't need to import this function because we want to use _your_ find peaks function